Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -242,7 +242,7 @@ - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController
 			default:
 				break;
 		}
-		
+		self.checkout = self.applePayAuthorizationDelegate.checkout;
```

Comment:
This change, while it looks benign, is leaking information from the delegate that the payment provider should not know about (that the checkout has changed).  A new delegate method implemented by the payment provider indicating the checkout has been updated is a possible solution.  @bgulanowski, thoughts?